src/main/java/frc/robot/commands/autonomous/DriveAuto.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands.autonomous;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Drivetrain;
+
+public class DriveAuto extends Command {
```

Comment:
perhaps its better to make this a command group and add the `Drive` command to it with a timeout